posthog/temporal/codec_server.py

```diff
@@ -0,0 +1,102 @@
+import json
+import base64
+
+from django.conf import settings
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.views.decorators.http import require_http_methods
+
+from temporalio.api.common.v1 import Payload
+
+from posthog.temporal.common.codec import EncryptionCodec
+
+
+@csrf_exempt
+@require_http_methods(["POST", "OPTIONS"])
+def decode_payloads(request):
```

Comment:
I know this is only enabled for local set ups right now, but can we add auth to this? https://docs.temporal.io/production-deployment/data-encryption#authorization